pallets/popi/src/tests.rs

```diff
@@ -4,24 +4,21 @@ use frame_support::{assert_noop, assert_ok};
 fn i_know_how_to_work_with_vectors() {}
 
 #[test]
-fn it_works_for_default_value() {
+fn correct_error_for_none_value() {
 	new_test_ext().execute_with(|| {
-		// Go past genesis block so events get deposited
-		System::set_block_number(1);
-		// Dispatch a signed extrinsic.
-		assert_ok!(Popi::do_something(RuntimeOrigin::signed(1), 42));
-		// Read pallet storage and assert an expected result.
-		assert_eq!(Popi::something(), Some(42));
-		// Assert that the correct event was deposited
-		System::assert_last_event(Event::SomethingStored { something: 42, who: 1 }.into());
+		// Ensure the expected error is thrown when no value is present.
+		assert_noop!(Popi::cause_error(RuntimeOrigin::signed(1)), Error::<Test>::NoneValue);
 	});
 }
 
 #[test]
-fn correct_error_for_none_value() {
+fn interactions_should_be_unique() {
 	new_test_ext().execute_with(|| {
-		// Ensure the expected error is thrown when no value is present.
-		assert_noop!(Popi::cause_error(RuntimeOrigin::signed(1)), Error::<Test>::NoneValue);
+		assert_ok!(Popi::interact(RuntimeOrigin::signed(1), 2, 1, 1));
```

Comment:
@DeluxeRaph for now I'm hardcoding ids of project and task. It would be great to have a functionality to register a board, and to register a task to a speciffic board.

Reply:
I think thats a great idea. This makes it clear the work work that needs to be done, and it shows where our efforts show be placed

Reply:
What is a board? Is this a place where users can select the task they want to work on?

Reply:
Yes, in agile it is for example the kanban board, the scrum board. This place where there is some overview of the tasks to be worked.